examples/telemetry/prometheus.yaml

```diff
@@ -61,6 +61,12 @@ data:
     - job_name: "serviceweaver"
       kubernetes_sd_configs:
       - role: pod
+      relabel_configs:
```

Comment:
can we add a comment or a pointer to the stack overflow link? so we know what's going on here?